editoast/src/views/timetable.rs

```diff
@@ -64,17 +66,107 @@ use crate::AppState;
 use crate::error::Result;
 use crate::models;
 use crate::models::Infra;
-use crate::models::paced_train::OccurrenceId;
 use crate::models::paced_train::PacedTrainChangeset;
-use crate::models::paced_train::TrainId;
 use crate::models::timetable::Timetable;
 use crate::models::timetable::TimetableWithTrains;
 use crate::models::train_schedule::TrainScheduleChangeset;
 use crate::views::AuthenticationExt;
 use crate::views::AuthorizationError;
+use crate::views::timetable::paced_train::iter_occurrences;
 use crate::views::timetable::simulation::SimulationResponseSuccess;
 use editoast_models::prelude::*;
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+#[serde(tag = "type")]
+pub(in crate::views::timetable) enum OccurrenceId {
+    BaseOccurrence { index: u64 },
+    ModifiedException { index: u64, exception_key: String },
+    CreatedException { exception_key: String },
+}
+
+#[derive(Debug, Clone, PartialEq)]
+/// This ID is used to identify paced train occurrences and exceptions when sending them to the core API for conflict detection.
```

Comment:
```suggestion
/// This ID is used to identify paced train occurrences and exceptions
```

I don't think this is useful how it is used, mostly what it is. Also, mentioning that it is used for `core` is weird in `views`.

Reply:
Done.